corehq/apps/sms/views.py

```diff
@@ -941,10 +942,19 @@ def backend_classes(self):
 
     def _fmt_backend_data(self, backend):
         is_editable = not backend.is_global and backend.domain == self.domain
+        if len(backend.supported_countries) > 0:
+            if backend.supported_countries[0] == '*':
+                supported_countries = _('Multiple%s') % '*'
```

Comment:
not really, was debating which way would make it look nicer for translating and just went with this way